test/parallel/test-child-process-stdio.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var options = {stdio: ['pipe']};
-var child = common.spawnPwd(options);
+let options = {stdio: ['pipe']};
+let child = common.spawnPwd(options);
 
 assert.notEqual(child.stdout, null);
 assert.notEqual(child.stderr, null);
```

Comment:
@pmatzavin Sorry for the late reply! Yes, that’s what I meant. Just to be clear, `notStrictEqual` corresponds to `!==` and `strictEqual` to `===`, and we usually do want the strict tests exactly to catch things like accidentally relying on  `undefined == null`.